app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
```

Comment:
remove, please, "#write your code here" and "pass" keyword

Reply:
Ostap, I don't understand, what are you talking about. Here is screen of this solution on my page at git_hub. There is no "#write your code here" or "pass" 
![Screenshot 2023-02-06 at 22 57 05](https://user-images.githubusercontent.com/122725491/217084187-f58016e4-f938-4728-b548-b018385b4f1d.png)